tensorflow/python/ops/logging_ops.py

```diff
@@ -149,6 +160,11 @@ def image_summary(tag, tensor, max_images=3, collections=None, name=None):
   return val
 
 
+@deprecated(
+    "2016-11-30", "Please switch to tf.summary.audio. Note that "
+    "tf.summary.histogram uses the node name instead of the tag. "
```

Comment:
Good catch. Looksl ike this branch is just bringing all the r12 changes into master, so maybe I should fix this in a separate PR to master.